webapp/components/Registration/CreateUserAccount.vue

```diff
@@ -109,30 +120,40 @@ export default {
       disabled: true,
       success: null,
       backendErrors: null,
+      checkedConfimed: false,
     }
   },
   props: {
     nonce: { type: String, required: true },
     email: { type: String, required: true },
   },
   methods: {
+    checked: function() {
+      this.backendErrors = { message: null }
```

Comment:
I would not misuse `backendErrors`. As the name implies, backend errors are for backend errors. You want to show a frontend error.